home/api.py

```diff
@@ -174,12 +179,46 @@ def get_queryset(self):
         return queryset
 
 
+class WhatsAppTemplateViewset(BaseAPIViewSet):
+    model = WhatsAppTemplate
+    base_serializer_class = WhatsAppTemplateSerializer
+    listing_default_fields = BaseAPIViewSet.listing_default_fields + [
+        "name",
+        "body",
+    ]
+    known_query_parameters = BaseAPIViewSet.known_query_parameters.union(["page", "qa"])
```

Comment:
I don't think `page` is a query param we use here, or at least it's not being handled in `get_queryset`

Reply:
Cool, removed it